docs/ref/contrib/staticfiles.txt

```diff
@@ -285,11 +285,17 @@ following requirements are met:
 * the :setting:`STATICFILES_STORAGE` setting is set to
   ``'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'``
 * the :setting:`DEBUG` setting is set to ``False``
-* you use the ``staticfiles`` :ttag:`static<staticfiles-static>` template
-  tag to refer to your static files in your templates
 * you've collected all your static files by using the
   :djadmin:`collectstatic` management command
 
+.. versionchanged:: 1.10
+
+    * you no longer have to use ``staticfiles`` :ttag:`static<staticfiles-static>`
+      template tag to refer to your static files in your templates.
+      The ``static`` template tag uses the ``staticfiles`` template tag,
```

Comment:
Ah. That's…awkward.

I think that `{% load static %}` can be reasonably documented *here*, in the `staticfiles` documentation, even though it's not actually part of it; it would also be a good place to explain why this is the case (which is then a hint to 3PA authors, and could be cross-refed from the 3PA primer). Then a `versionchanged` tag to note that prior to <1.10> this didn't work like that.

However I note that the staticfiles documentation is…poor at the moment. So perhaps you could do something accurate but terse here, and then further work at some point can improve things. (Depends how much time you have right now, really.)